webapp/app/js/models/resultsets_store.js

```diff
@@ -852,16 +852,22 @@ treeherder.factory('ThResultSetStore', [
         return _.reduce(
             jobList,
             function(memo, job){
-                if(job.state == "pending" || job.state == "running"){
-                    memo[job.state]++;
-                }else{
+                // if a job is completed, count the result instead of the state
+                if(job.state === "completed"){
                     memo[job.result]++;
                 }
+
+                // count these as coalesced, not as pending
+                if (job.job_coalesced_to_guid) {
+                    memo.coalesced++;
+                } else {
+                    memo[job.state]++;
```

Comment:
Do we still need to count the individual results? We're only using "still outstanding" vs "total", right? (with coalesced ignored from both groups)